### DISCUSSION TITLE
Flutter Web Email Attachment as PDF

### DISCUSSION THREAD

**Discussion by faizan1249** [community/community#42356] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Can Someone please help me out regarding Email Attachment in flutter Web. I have already done this thing easily in the Android Using the path_provider to first store it in the Temp Directory and then getting the path to share it on the email or anywhere. But Unfortunately path_provider doesn't support in web. So I want that same thing in flutter web. My PDF file is converted into Bytes. How can I Attach that file with the Email in windows or Mac using Flutter Web.

I have Tried share_plus and path_provider in web but path_provider doesn't support in web similarly dart:io.

**Comment by queenofcorgis** [comment]
Hi @faizan1249 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Flutter. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
